packages/governance-app/src/Council/CouncilMembers.tsx

```diff
@@ -28,11 +30,22 @@ export function CouncilMembers () {
       {
         activeCouncil && activeCouncil.map(([accountId, blockNumber]: [AccountId, BlockNumber]) => {
           let name;
-          try {
-            name = keyring.getAccount(accountId).getMeta().name;
-          } catch (e) {
-            name = '';
-          }
+
+          tryCatch2v(
+            () => keyring.getAccount(accountId),
+            (e: any) => new Error(e.message)
+          )
+          .chain((pair) => tryCatch2v(
+            () => {
+              if (!isUndefined(pair)) {
+                return pair.meta;
```

Comment:
name = pair.meta.name
```suggestion
                return pair.meta;
```